src/OpenConext/ProfileBundle/Resources/views/MyConnections/active-connections.html.twig

```diff
@@ -0,0 +1,45 @@
+{% set activeConnections = {
```

Comment:
Was this deliberate? If this was not for testing purposes  the activeConnections is empty check below could be removed else the set should be removed.

Reply:
good catch x2!